client/src/containers/scenarios/new/index.tsx

```diff
@@ -51,7 +99,35 @@ const ScenariosNewContainer: React.FC = () => {
         </div>
       </form>
       <div className="h-full flex flex-col w-auto bg-white py-6 overflow-auto relative">
-        <InterventionsAttributes handleNewIntervention={handleNewIntervention} />
+        <div className="relative">
+          <div className="flex items-center content-between text-sm">
+            <div className="pb-6 flex items-center justify-between w-full">
+              <div className="space-x-2">
+                <button
+                  type="button"
+                  className={cx({ 'border-b-2 border-green-700': interventionsContent })}
+                  onClick={() => handleTab('interventions')}
+                >
+                  Interventions ({interventions.length})
+                </button>
+
+                <button
+                  type="button"
+                  className={cx({ 'border-b-2 border-green-700': !interventionsContent })}
+                  onClick={() => handleTab('growth')}
+                >
```

Comment:
```suggestion
                <button
                  type="button"
                  className={cx({ 'border-b-2 border-green-700': !interventionsContent })}
                  onClick={handleTab}
                  dataTab="growth"
                >
```
Using data aria attributes you can pass values and not to reassign the `onClick` in every render loop